packages/components-react/src/heading-group/HeadingGroup.tsx

```diff
@@ -0,0 +1,11 @@
+import {
+  type HeadingGroupProps,
+  HeadingGroup as UtrechtHeadingGroup,
+} from '@utrecht/component-library-react/dist/css-module';
+import { PropsWithChildren, ReactNode } from 'react';
+
+export const LuxHeadingGroup = ({ children, ...restProps }: PropsWithChildren<HeadingGroupProps>): ReactNode => {
+  return <UtrechtHeadingGroup {...restProps}>{children}</UtrechtHeadingGroup>;
+};
```

Comment:
Is het niet makkelijker om dit te doen? Want je doet geen verdere aanpassing aan Utrecht's HeadingGroup.

```suggestion
HeadingGroup.displayName = 'LuxHeadingGroup';

export const LuxHeadingGroup = HeadingGroup;
```
Kan de `as` bij de `import` ook weg.